AC-Music.py

```diff
@@ -1,81 +1,10 @@
+import os
 import datetime
 from playsound import playsound
-currentDT = datetime.datetime.now()
```

Comment:
This gets set every time we go through the main loop (lines 7-8), so there's no need to set them before we enter the loop.